detector/fcal/BeamCal_o1_v01_geo.cpp

```diff
@@ -191,17 +203,16 @@ static DD4hep::Geometry::Ref_t create_detector(DD4hep::Geometry::LCDD& lcdd,
           //std::cout << e.what()  << std::endl;
         }
 
+        // Check if a separate outer_radius is declared.
         double outerR = bcalOuterR;
         try {
-          const std::string& sliceType = compSlice.attr< std::string >(_Unicode(layerType));
-          if ( sliceType.compare("graphiteShielding") == 0) {
-            outerR = compSlice.outer_radius();
-          }
+          outerR = compSlice.outer_radius();
         } catch (std::runtime_error &e) {
               //std::cout << "Catching " << e.what()  << std::endl;
               //std::cout << e.what()  << std::endl;
         }
 
+        // std::cout << "Layer R_out = " << outerR << std::endl;
```

Comment:
Lines like this are exactly the kind of things that wouldn't happen with `git add -p`